### PR TITLE
feat: Use HTTP1 for service recovery and load balancing

### PR DESCRIPTION
**Pull request by ikhemissi**
This change consists on dropping HTTP2 in favour of HTTP1 in order to resolve 2 issues:
- Recover connection after the service recovers: this can also be achieved by re-attempting connections and handling extra HTTP2 headers like `GOAWAY` (in addition to the current `ERR_HTTP2_INVALID_SESSION`).
- Load balancing requests between multiple instances of the same service: HTTP2 creates a session which binds 2 services (instances) together. As a session-less protocol, HTTP1 does not have this issue, but on the other hand, there will be a handshake every time a request is made.

BREAKING CHANGE: HTTP2 is dropped and replaced with HTTP1 (without SSL/TLS).

### CODE REVIEW

File: respondWith.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	if (data instanceof Error) {
+		const {name, message, stack, code} = data;
+		return JSON.stringify({name, message, stack, code});
+	}
+
+	return JSON.stringify(data);
+}
+
+function respondWith({response, data, headers = {}, statusCode = 200}) {
+	// console.log('respondWith', {data, statusCode});
```
Comment (cboyce183) [inline respondWith.js]: Naughty leaving commented out `console.log()`

File: sendJson.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+			reject(error);
+		});
+
+		request.write(data);
+		request.end();
+	});
+}
+
+async function sendJson({data, port, hostname = 'localhost', path = '/', apiKey = ''}) {
+	// console.log('sendJson', {hostname, port, path, data, apiKey});
```
Comment (cboyce183) [inline sendJson.js]: Naughty

**Comment by ikhemissi** [comment]
This package was published here: https://www.npmjs.com/package/@5app/buslane

**Comment by ikhemissi** [comment]
it starts with the version `3.00` instead of `1.0.0` to avoid confusions (as the previous package was published under version 2).

**Pull request merged**